packages/lit-dev-content/site/blog/2022-10-13-lit-labs-event.md

```diff
@@ -0,0 +1,38 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Get the low-down on Lit Labs"
+summary: "We are having a virtual event on October 24th 2022 at 10 AM PST to talk about the status of Lit Labs!"
+date: 2022-10-13
+author:
+  - elliott-marquez
+eventDate: "Monday October 24th 2022 at 10AM PDT"
+ytLink: https://www.youtube.com/
+---
+
+The Lit team is hosting a virtual event on **{{eventDate}}** to talk about the status of Lit Labs. Join us for detailed presentations on the current and future offerings of Lit labs, their statuses, and how you can help get them across the line and shape the future of Lit!
```

Comment:
Nit: My initial reaction to "talk about the status of Lit Labs" sounded sorta ominous, along the lines of "let's talk about the status of our relationship" as if something bad is about to drop. 

Maybe something a little more obviously positive, like

```suggestion
The Lit team is hosting a virtual event on **{{eventDate}}** to talk about all of the exciting work going on in Lit Labs. Join us for detailed presentations on the current and future offerings of our Labs packages, their statuses, and how you can help get them across the line and shape the future of Lit!
```